alfresco-dynamic-extensions-repo/build.gradle

```diff
@@ -39,9 +39,14 @@ subprojects {
         ampLib "biz.aQute.bnd:biz.aQute.bnd:${project.ext.bndVersion}"
 
         blueprint(project(path: ":blueprint-override", configuration: "patchBlueprint${project.ext.geminiVersion.substring(0, 1)}"))
-        blueprint(project(":blueprint-integration"))
-        blueprint(project(":blueprint-integration:blueprint-integration-spring-3"))
-        blueprint(project(":blueprint-integration:blueprint-integration-spring-5"))
+
+        if(project.name.contains("-5")) {
+            ampLib("org.springframework:spring-core:${project.ext.springVersion}") { transitive = false }
```

Comment:
Why are we putting a spring package inside the AMP?

Reply:
I thought it was needed, but apparently not.